### ISSUE TITLE
Geospatial Lesson 5 Issue

### ISSUE THREAD

**Issue by ddlj1397** [enomis-dev/YouTube#2]
Hello,

Thanks for this tutorial, it is very helpful. I currently have a question during the training step of this .ipynb., specifically at:

`loss = criterion(outputs, targets.squeeze(1))`

The errors I am getting is that certain target values are out of bounds. For example "Target 82 is out of bounds".  I am wondering if you had similar issues with different range of values, and if so how did you resolve those issues?
![image](https://github.com/user-attachments/assets/325c73da-04cf-4ba0-b646-c76b3ae9ebfe)

**Comment by enomis-dev** [comment]
Hi @ddlj1397 thank you for your support. I didn't find this error while preparing the notebook, did you change anything in the previous sections? Perhaps you could print the shape and the values of target to have further information to debug.

**Comment by ddlj1397** [comment]
@enomis-dev Thanks for you response and my apologies for responding so late. I ended up changing the class values to '89' and that fixed the issue. But I am not sure why I was not able to get it to work with 13 classes.
